internal/helm/chart/builder_local.go

```diff
@@ -189,14 +189,17 @@ func (b *localChartBuilder) Build(ctx context.Context, ref Reference, p string,
 // mergeFileValues merges the given value file paths into a single "values.yaml" map.
 // The provided (relative) paths may not traverse outside baseDir. It returns the merge
 // result, or an error.
-func mergeFileValues(baseDir string, paths []string) (map[string]interface{}, error) {
+func mergeFileValues(baseDir string, paths []string, ignoreMissing bool) (map[string]interface{}, error) {
 	mergedValues := make(map[string]interface{})
 	for _, p := range paths {
 		secureP, err := securejoin.SecureJoin(baseDir, p)
 		if err != nil {
 			return nil, err
 		}
 		if f, err := os.Stat(secureP); err != nil || !f.Mode().IsRegular() {
```

Comment:
we should still fail when not a regular file.

Reply:
I think I've covered this now.